### DISCUSSION TITLE
<Uploading code from desktop, then setting up codebase for engineers>

### DISCUSSION THREAD

**Discussion by zachbond007** [community/community#49798] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

New to engineering. Bringing a new team on board to update our app. I need to upload my existing code (from desktop) into Git-Hub, then need to set up a code space so new team can edit/update as needed. What are the step by step instructions to do this?

Please help
Thanks 
Zach

**Comment by 03timothyash** [comment]
Va ser un gusto trabajar contigo. Saludos

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: zachbond007 ***@***.***>
Sent: Saturday, March 11, 2023 8:12:04 AM
To: community/community ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GitHub Community] <Uploading code from desktop, then setting up codebase for engineers> (Discussion #49798)


Select Topic Area

Question

Body

New to engineering. Bringing a new team on board to update our app. I need to upload my existing code (from desktop) into Git-Hub, then need to set up a code space so new team can edit/update as needed. What are the step by step instructions to do this?

Please help
Thanks
Zach

**Comment by enviGit** [comment]
Hi Zach!

Here are the step-by-step instructions for uploading your existing code to GitHub and setting up a code space for your new team to work on:
- **Create a new repository:** Once you're signed in, create a new repository by clicking on the "New" button on your GitHub dashboard. Give your repository a name and description, and make it public or private as desired.
- **Upload your code:** Once your repository is created, you can upload your existing code by clicking on the "Upload files" button on the repository's main page. You can either drag and drop your code files into the upload area or select them manually.
- **Commit your changes:** After you've uploaded your code, you'll need to commit your changes to the repository. To do this, enter a commit message describing the changes you made and click the "Commit changes" button.
- **Invite team members:** Now that your code is on GitHub, you can invite your new team members to collaborate on the project. Click on the "Settings" button in your repository and select "Manage access". From there, you can invite team members by entering their GitHub usernames or email addresses.
- **Set up a code space:** To allow your team members to edit and update the code, you'll need to set up a code space. A code space is a cloud-based development environment that allows multiple people to work on the same code at the same time. To set up a code space, go to the "Code" tab in your repository and click the "New code space" button. Follow the prompts to set up your code space.
- **Share your code space:** Once your code space is set up, you can share it with your team members. Click on the "Share" button in your code space and enter the email addresses of the people you want to invite. They'll receive an email with a link to access the code space.

That's it! Your team members can now access your code on GitHub and work on it using the code space you set up.
